effekt/shared/src/main/scala/effekt/symbols/symbols.scala

```diff
@@ -197,6 +197,7 @@ enum Binder extends TermSymbol {
   case RegBinder(name: Name, tpe: Option[ValueType], region: BlockSymbol, decl: RegDef) extends Binder, RefBinder
   case VarBinder(name: Name, tpe: Option[ValueType], decl: VarDef) extends Binder, RefBinder, TrackedParam
   case DefBinder(name: Name, tpe: Option[BlockType], decl: DefDef) extends Binder, BlockSymbol
+  case ConstBinder(name: Name, tpe: Option[ValueType], decl: ConstDef) extends Binder, ValueSymbol
```

Comment:
Would it work to introduce them as a `BlockSymbol` (morally nullary functions)? This confuses people because they are introduced with the keyword `val`.